packages/core/src/config/config-resolvers.ts

```diff
@@ -407,6 +412,24 @@ function groupStyleguideAssertionRules(
   for (const [ruleKey, rule] of Object.entries(rules)) {
     if (ruleKey.startsWith('assert/') && typeof rule === 'object' && rule !== null) {
       const assertion = rule;
+      if (plugins) {
+        for (const field of Object.keys(assertion)) {
+          const [pluginId, fn] = field.split('/');
+          if (!pluginId || !fn) continue;
+          const plugin = plugins.find((plugin) => plugin.id === pluginId);
+          if (!plugin) {
+            throw Error(`Plugin ${colorize.red(pluginId)} isn't found.`);
```

Comment:
What do you think about this: 
```suggestion
            throw Error(colorize.red(`Plugin ${colorize.blue(pluginId)} isn't found.`));
```
?